### ISSUE TITLE
Content not updating in some cases since 4.0.12 in Chrome

### ISSUE THREAD

**Issue by volkv** [froala/wysiwyg-editor#4493]
When you have in editor text like `word word word word word` and then put cursor after letter `d`, then press space and write a `new word`, then editor's content is not save (`contentChanged` event also is not fired). Firefox is fine. Chrome <= 4.0.11 is also fine.

I guess it's related to this cursor jump issue https://github.com/froala/wysiwyg-editor/issues/4463

And `ZeroWithSpace` character which appeared in 4.0.12+

**Comment by nategadzhi** [comment]
Yup. We've started running into this on our production, but for us, ironically, it triggered on Chrome on 4.0.13, and only when we're using react-froala-wysiwyg. If we mount Froala manually, without the react wrapper, it.... works. I think it's because we're manually binding onchange and onBlur and save the content from the editor. 

Any updates from the core team on when to expect a fix?

**Comment by volkv** [comment]
It's just another 846th issue ticket. No much hope for this

**Comment by ilyaskarim** [comment]
We apologize for the inconvience caused, This is one of the top priority issues and the fix will be available in the V4.0.15 editor release.

**Comment by nategadzhi** [comment]
Hey @ilyaskarim, thanks for the update! 
It sounds like 4.0.14 should contain a few fixes that are very important for my org. @lateworm reached out recently over email, too. 

Any updates on when to expect the release? Looking at the release cadence, seems like it should be out soon?

**Comment by ilyaskarim** [comment]
Hi @nategadzhi, That's correct, We have some important fixes coming in V4.0.14,  We are expecting the release by late this week.

**Comment by Abildtoft** [comment]
Was this fixed in V4.0.14?

**Comment by sanderdejongg** [comment]
Hi everyone, 

If you're still facing this issue a bit of a hacky way to get around the content changed event is to bind a keydown event to the editor and fetch the html using "editor.html.get()".

Happy coding!

**Comment by zahidkarim24** [comment]
Issue fixed in V4.0.15: https://froala.com/wysiwyg-editor/changelog/#4.0.15
